internal/frontend/proto/tesseral/frontend/v1/frontend.proto

```diff
@@ -304,6 +306,11 @@ service FrontendService {
   rpc ListAPIKeyRoleAssignments(ListAPIKeyRoleAssignmentsRequest) returns (ListAPIKeyRoleAssignmentsResponse) {
     option (google.api.http) = {get: "/frontend/v1/api-key-role-assignments"};
   }
+
+  // Lists audit log events for an organization.
```

Comment:
nix comment for now, we'll do them all in one pass. reason being, it's tied up in our SDK codegen pipeline and I'd prefer consistency